### PR TITLE
Use greater precision when persisting float values

### PR DESCRIPTION
**Pull request by ph1lm**

### CODE REVIEW

**Comment by karkum** [comment]
@ph1lm can you please update release notes? And fix up some checkstyle issues? Thanks!

**Comment by ph1lm** [comment]
@karkum All comments are addressed.

**Comment by karkum** [comment]
This looks good to me! :shipit:

File: src/main/java/com/opower/persistence/jpile/infile/InfileDataBuffer.java [hunk 1]
```diff
@@ -195,6 +199,26 @@ public final InfileRow append(byte[] bytes) {
...
+        String fractional = Strings.repeat("#", scale);
+
+        DecimalFormatSymbols decimalFormatSymbols = new DecimalFormatSymbols();
+        decimalFormatSymbols.setDecimalSeparator('.');
+        DecimalFormat decimalFormat = new DecimalFormat(integer + "." + fractional);
+        decimalFormat.setGroupingUsed(false);
+        decimalFormat.setDecimalFormatSymbols(decimalFormatSymbols);
+
+        String formatted = decimalFormat.format(number);
+        append(formatted);
```
Comment (amir20) [inline src/main/java/com/opower/persistence/jpile/infile/InfileDataBuffer.java]: `return append(formatted);` instead?

Comment (ph1lm) [inline src/main/java/com/opower/persistence/jpile/infile/InfileDataBuffer.java]: Done

**Pull request merged**